engine/frontend/src/api/enclave.js

```diff
@@ -26,53 +23,43 @@ const createApiPromiseClient = (apiClient) => {
     return "";
 }
 
-export const makeRestApiRequest = async ( url, data, config) => {
+export const makeRestApiRequest = async (url, data, config) => {
     const response = await axios.post(`${ENGINE_URL}/${url}`, data, config)
     return response;
 }
 
-export const getEnclavesFromKurtosis = async () => {
-    const respFromGrpc = await makeRestApiRequest(
-         "engine_api.EngineService/GetEnclaves",
-        {"field":""},
-        {"headers":{'Content-Type': "application/json"}}
-    )
-
-    const {data} = respFromGrpc
-
+export const getEnclavesFromKurtosis = async (token) => {
+    const data = await getEnclavesFromEnclaveManager(token);
     if ("enclaveInfo" in data) {
         return Object.keys(data.enclaveInfo).map(key => {
             const enclave = data.enclaveInfo[key]
             return {
                 uuid: enclave.enclaveUuid,
                 name: enclave.name,
-                created: enclave.creationTime,
+                // created: enclave.creationTime,
                 status: enclave.apiContainerStatus,
-                apiClient: createApiPromiseClient(enclave.apiContainerHostMachineInfo)
+                host: enclave.apiContainerHostMachineInfo.ipOnHostMachine,
+                port: enclave.apiContainerHostMachineInfo.grpcPortOnHostMachine,
             }
-        })
+        });
     }
-
     return []
 }
 
-export const createEnclave = async () => {
-    const data = {
-        apiContainerVersionTag: "",
-        apiContainerLogLevel: "info",
-        isPartitioningEnabled: false,
-    }
-    const response = await makeRestApiRequest("engine_api.EngineService/CreateEnclave", JSON.stringify(data), {"headers":{'Content-Type': "application/json"}})
-
-    const enclave = response.data.enclaveInfo;
-    const apiClient = createApiPromiseClient(enclave.apiContainerHostMachineInfo);
+export const createEnclave = async (token) => {
+    const enclaveName = ""; // TODO We could make this input from the UI
+    const apiContainerVersionTag = "";
+    const apiContainerLogLevel = "info";
+    const response = await createEnclaveFromEnclaveManager(enclaveName, apiContainerLogLevel, apiContainerVersionTag, token)
 
+    const enclave = response.enclaveInfo;
     return {
         uuid: enclave.enclaveUuid,
         name: enclave.name,
         created: enclave.creationTime,
         status: enclave.apiContainerStatus,
-        apiClient
+        host: enclave.apiContainerHostMachineInfo.ipOnHostMachine,
```

Comment:
Just for book keeping; there's a bug which can be tackled in next pr. The host and port is only available for running enclaves - for stopped enclaves it will error out as it's not gonna be present.